dist/index.html

```diff
@@ -10,6 +10,12 @@
       href="https://fonts.googleapis.com/css2?family=Outfit:wght@400;500;800;900&display=swap"
       rel="stylesheet"
     />
+    <style>
+      .button button_primary_light{
```

Comment:
Class selector is '.button_primary_light' not 'button_primary_light'. Please fix 😉😉